x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/render_cell_value.tsx

```diff
@@ -90,9 +90,11 @@ export function getAlertFormatters(fieldFormats: FieldFormatsRegistry) {
           </>
         );
       case ALERT_ANOMALY_SCORE:
+        const resultValue: number[] = typeof value === 'number' ? [value] : value;
+        const latestValue = resultValue.at(-1) as number;
```

Comment:
Is this correct? Looks like it is rendering the last digit rather than the last value in the array e.g. `5` is rendered for `1.2345`.

Also we should probably use the same approach to formatting as elsewhere, which I believe floors the anomaly score and then renders with no decimal places.

Reply:
Formatting with `getFormattedSeverityScore` looks good.